modules/rosa-initialize.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/get-started-with-rosa.adoc
+
+[id="rosa-initialize_{context}"]
+= Initializing {product-title}
+
+Use the `init` command to initialize {product-title} (ROSA).
+
+[id="rosa-init_{context}"]
+== init
+
+Performs a series of checks to verify that you are ready to deploy an {product-title} cluster.
+
+The list of checks includes the following:
+
+* Checks to see that you have logged in (see `login`)
+* Checks that your AWS credentials are valid
+* Checks that your AWS permissions are valid (see `verify permissions`)
+* Checks that your AWS quota levels are high enough (see `verify quota`)
+* Runs a cluster simulation to ensure cluster creation will perform as expected
+* Checks that the osdCcsAdmin user has been created in your AWS account
+* Checks that the OpenShift command-line tool is available on your system
+
+.Syntax
+[source,terminal]
+----
+$ rosa init [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--region
+|The AWS region (string) in which verify quota and permissions. This value overrides the `AWS_REGION` environment variable.
+
+|--delete-stack
+|Deletes stack template applied to your AWS account during the `init` command.
+
+|--client-id string
+|The OpenID client identifier (string). Default: `cloud-services`
+
+|--client-secret
+|The OpenID client secret (string).
+
+|--insecure
+|Enables insecure communication with the server. This disables verification of TLS certificates and host names.
+
+|--scope
+|The OpenID scope (string). If this option is used, it completely replaces the default scopes. This can be repeated multiple times to specify multiple scopes. Default: `openid`
+
+|--token
+|Accesses or refreshes token (string).
+
+|--token-url
+|The OpenID token URL (string). Default: `https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token`
```

Comment:
Just checking whether this is okay to be a clickable URL, or if it should be escaped? I get an error when I click it.

Reply:
It should be a literal value. I just escaped it with `\ and it seems to be publishing correctly now.